src/adapters/chineseloc/chinese-source/chinese-source.js

```diff
@@ -256,7 +267,7 @@ export default class ChineseSource {
                   if (k + 1 < pin.length - 1 && pin[k + 1] === ':') { pin = pin.replace('u:', _v[tone]) } else { pin = pin.replace('u', _u[tone]) }
                   break
                 default:
-                  console.info('some kind of weird vowel')
+                  console.warn('some kind of weird vowel')

```

Comment:
Maybe it's better to print a vowel and probably its code here so we could analyze it?

Reply:
It is from the previous Alpheios 1.0 version, so I didn't deeply analyze what vowel is meant here.
Simply got this code from there.
I suggest to postpone it to the non-prototype version of the CEDICT

Reply:
ok